basic/battery_voltage.py

```diff
@@ -0,0 +1,13 @@
+# Battery pin is board.BATTERY (GPIO9)
+import time
+import board
+from analogio import AnalogIn
+
+analog_in = AnalogIn(board.BATTERY)
+
+def get_voltage(pin):
+    return (pin.value * 3.3) / 65536
+
```

Comment:
Divided by 2 is missing (100k/100k voltage divider) but it seems to be connected to VBUS instead of VBATTERY

Reply:
Bug report on LilyGO github account is planned